guides/README.md

```diff
@@ -128,18 +133,11 @@ Some files are included in different contexts, there are attributes to find the
 
 When doing review, consider checking out the topic branch and putting necessary changes on top of author's work to making many comments on github.
 
-We should avoid using phases like, "Starting from version 6.5 or 1.22" because it is not possible to easily translate these strings in both the streams.
-
-## Structure
+Avoid using phases like "Starting from version 6.5 or 1.22" because it is not possible to easily translate these strings in both the streams.
```

Comment:
What does "both" refer to? RH Satellite & Foreman? If so, maybe say something like "all streams".